### DISCUSSION TITLE
syntax error, unexpected 'static' (T_STATIC) in SortableTrait.php

### DISCUSSION THREAD

**Discussion by kailashkbs** [freekmurze/eloquent-sortable#122] [Q&A]
I am getting this error after successful installation.
  During class fetch: Uncaught Parse Error: syntax error, unexpected 'static' (T_STATIC) in /vendor/spatie/eloquent-sortable/src/SortableTrait.php:82 Stack trace: #0 .
  
  Thanks.

**Comment by freekmurze** [comment]
The package does not contain a syntax error. You're probably using the wrong PHP version.

**Reply by kailashkbs** [reply]
i am using PHP version 8.0. is there any compatibility issue with this version?

**Reply by freekmurze** [reply]
No, see: https://github.com/spatie/eloquent-sortable/actions/runs/1878695078
